dashboard/src/components/SideMenu/SideMenu.tsx

```diff
@@ -68,19 +72,22 @@ const SideMenu = (): JSX.Element => {
 
   const items: MenuItems[] = [
     {
+      id: 0,
```

Comment:
we can just use the item index for this purpose

Reply:
we should use the path url for this